bot/seasons/evergreen/fun.py

```diff
@@ -15,22 +15,14 @@ def __init__(self, bot):
         self.bot = bot
 
     @commands.command()
-    async def roll(self, ctx, str_input: str = None):
+    async def roll(self, ctx, num_rolls: int):
         output = ""
-
-        if not str_input:
-            output = "To use .roll, format it as such: .roll (number)"
-        else:
-            try:
-                num_rolls = int(str_input)
-                if num_rolls > 6:
-                    num_rolls = 6
-                elif num_rolls < 1:
-                    return
-                for i in range(num_rolls):
-                    output += ":terning%d: " % random.randint(1, 6)
-            except ValueError:
-                return
+        if num_rolls > 6:
+            num_rolls = 6
+        elif num_rolls < 0:
```

Comment:
Shouldn't this still be `< 1`?

Reply:
I was over engineering it and thought "well maybe they don't want to roll anything" and then realized I'm just being obtuse.